pkgs/applications/audio/musikcube/default.nix

```diff
@@ -0,0 +1,57 @@
+{ cmake
+, pkgconfig
+, alsaLib
+, boost
+, curl
+, fetchFromGitHub
+, ffmpeg
+, lame
+, libev
+, libmicrohttpd
+, ncurses
+, pulseaudio
+, stdenv
+, taglib
+, systemdSupport ? stdenv.isLinux, systemd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "musikcube";
+  version = "0.83.0";
+
+  src = fetchFromGitHub {
+    owner = "clangen";
+    repo = pname;
+    rev = version;
+    sha256 = "0xrsk23f8ds54w9zxm4ac3ldn0jrn1fgz49fnfbvmgqpqcyyd2dm";
+  };
+
+  postPatch = ''
+    sed -i 's|add_custom_command(TARGET postbuild POST_BUILD COMMAND ./strip-nix.sh)||' CMakeLists.txt
+  '';
```

Comment:
I prefer patches since they make it obvious when the replacement no longer applies.

Or maybe ask upstream about an option for disabling this.

Reply:
It should also be below `buildInputs`, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887

Reply:
As always thank you for the wonderful review @jtojnar. I hope what I have pushed meets your standards. I went ahead and made a request upstream as well: https://github.com/clangen/musikcube/issues/339.